d2bs/kolbot/libs/MuleLogger.js

```diff
@@ -27,7 +27,7 @@ var MuleLogger = {
 	LogEquipped: false, // include equipped items
 	LogMerc: false, // include items merc has equipped (if alive)
 	SaveScreenShot: false, // Save pictures in jpg format (saved in 'Images' folder)
-	IngameTime: 20, // Time to wait after leaving game
+	MulePerm: false, // perming the new created mule chars 2h in game
```

Comment:
I don't think we can get rid of the IngameTime option, people might want min game time to not get rd.